packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -176,6 +182,11 @@ export class MonacoEditorProvider {
 
     protected injectKeybindingResolver(editor: MonacoEditor): void {
         const keybindingService = editor.getControl()._standaloneKeybindingService;
+        if (!keybindingService) {
+            console.warn('Cannot inject Keybinding Resolver because KeybindingService is not recognized');
```

Comment:
Could it happen in the normal application that it is undefined? If not then we should fail, i think it is fine without a guard it should fail with undefined reference exception and it has to be fixed. Please remove such checks if there are more.

Reply:
I added such checks because `_standaloneKeybindingService` [became nullable](https://github.com/microsoft/vscode/commit/31c73988b9fe71ea10ebf881d0a178f3d8e14743) and VS Code [has similar checks](https://github.com/microsoft/vscode/blob/31c73988b9fe71ea10ebf881d0a178f3d8e14743/src/vs/editor/standalone/browser/standaloneCodeEditor.ts#L190).

I removed this checks for `theia`: https://github.com/eclipse-theia/theia/pull/7149/commits/02ae87b8f87849e4b787e2d78f461ea57c163848